recipes/pycallgraph2/meta.yaml

```diff
@@ -16,8 +16,6 @@ build:
 
 requirements:
   host:
-    - pip
```

Comment:
Oh.  You removed pip too! It should be only pytest. You can try that in the feedstock later.

Reply:
Sorry I misunderstood. Where can I try to change this?

Reply:
There will be a new repository, where you'll have access so you can send updates to this packages, that we call feedstock. You can test it later there. Watch out for the GitHub invite.